packages/frontend/pages/index.tsx

```diff
@@ -2,6 +2,7 @@ import { ConnectButton } from "@rainbow-me/rainbowkit";
 import type { NextPage } from "next";
```

Comment:
There are some prettier issues in this file, so the lint tests are failing.

![image](https://user-images.githubusercontent.com/2486142/197308049-e1b0e61b-0062-407a-bd1f-063e921288b7.png)